message_sender/models.py

```diff
@@ -76,3 +77,28 @@ class OutboundSendFailure(models.Model):
 
     def __str__(self):  # __unicode__ on Python 2
         return str(self.id)
+
+
+@python_2_unicode_compatible
+class Channel(models.Model):
+
+    VUMI_TYPE = 'vumi'
+    JUNEBUG_TYPE = 'junebug'
+
+    CHANNEL_TYPES = (
+        (JUNEBUG_TYPE, 'Junebug'),
+        (VUMI_TYPE, 'Vumi')
+    )
+
+    channel_id = models.CharField(unique=True, null=False, blank=False,
+                                  max_length=64)
+    channel_type = models.CharField(choices=CHANNEL_TYPES, max_length=20,
+                                    default=JUNEBUG_TYPE)
+    concurrency_limit = models.IntegerField(null=False, blank=False, default=0)
+    message_delay = models.IntegerField(null=False, blank=False, default=0)
+    message_timeout = models.IntegerField(null=False, blank=False, default=0)
+    default = models.BooleanField(default=False)
```

Comment:
The idea would be when you save a channel with `default=True`, it would make all other channels `default=False`, so that there would only be one default channel.

That way, if you want to change the default channel, make another channel defaut, and the old default channel will be marked as not default.